src/features/game/types/consumables.ts

```diff
@@ -10,6 +10,15 @@ type JuiceName =
   | "Power Smoothie"
   | "Bumpkin Detox";
 
+const Juices: JuiceName[] = [
```

Comment:
If we add a new smootie, is it possible that the developer forgets to add it here?

I would suggest using the `CONSUMABLES` constant and pulling off the names from there.

E.g. `getKeys(CONSUMABLES).filter(name => CONSUMABLES[name].building === 'Smoothie Shack'))`